### PR TITLE
Update the webpack chapter with new entrypoint example.

### PR DESCRIPTION
**Pull request by gabriel-glo**
This was needed for one of the recent projects and is a really useful thing.

### CODE REVIEW

**Review by dingo-d — DISMISSED** [review]
Cool. What was the use case for this?

**Review by iruzevic — APPROVED** [review]

File: website/docs/basics/webpack.md [hunk 1]
```diff
@@ -108,3 +108,24 @@ module.exports = (env, argv) => {
     ],
   };
 ```
+
+## Add a new entrypoint
+
+Let's say you want to add a new, separate JS and/or CSS build for some specific functionality. This functionality may or may not have anything to do with your WordPress theme or plugin, but it needs to be in separate files. To do that you'd need to add a new entrypoint for webpack to build:
```
Comment (IvanGrginovInf) [inline website/docs/basics/webpack.md:114]:
```suggestion
Let's say you want to add a new, separate JS and/or CSS build for some specific functionality. This functionality may or may not have anything to do with your WordPress theme or plugin, but it needs to be in separate files. To do that you'd need to add a new entrypoint for webpack to build in `webpack.config.js`:
```

**Review by IvanGrginovInf — CHANGES_REQUESTED** [review]
Left a comment, otherwise looks great!

**Comment by gabriel-glo** [comment]
@dingo-d I needed this to style some embedded components (in iframes) that allowed linking to the external stylesheets. So I separated the styles for this embed with this method.

**Review by IvanGrginovInf — APPROVED** [review]

**Pull request merged**